apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/DiagnosisUtilV2.java

```diff
@@ -130,15 +131,34 @@ static boolean containedIn(Diagnosis diag, CodeableConcept codeableConcept) {
    */
   static CodeableConcept toCodeableConcept(Diagnosis diag) {
     CodeableConcept codeableConcept = new CodeableConcept();
+    String system = diag.getFhirSystem();
+    String code = diag.getCode();
 
+    // Add an additional coding for CARIN conformance for ICD-10-cm
```

Comment:
This comment needs more explanation, I think. It should probably at least reference the Jira ticket.